packages/mkyara/PKGBUILD

```diff
@@ -0,0 +1,38 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=mkyara
+_pkgname=mkYARA
+pkgver=3.8147f91
+pkgrel=1
+pkgdesc='Tool to generate YARA rules based on binary code'
```

Comment:
Add end point to description.

Reply:
@Edu4rdSHL Is there a full list of packaging conventions somewhere that I can read?